ignite/handlers/timing.py

```diff
@@ -1,6 +1,10 @@
-import time
 from ignite.engine import Events
 
+try:
```

Comment:
nit: `time.time` is available in both python 2.7 and 3 so could just use that for both

Reply:
Yeah, but I thought that if `perf_counter` has higher precision, why not use it when available.

Reply:
Fair enough.